packages/node-mimimi/src/importer.rs

```diff
@@ -780,7 +777,7 @@ impl Importer {
 				let too_big_chunk_path = import_error
 					.path
 					.expect("All too bug chunk error should contain path");
-				FileImport::move_failed_eml_file(&PathBuf::from(too_big_chunk_path)).unwrap();
+				FileImport::move_failed_eml_file(&PathBuf::from(too_big_chunk_path)).ok();
```

Comment:
did you mean to replace `unwrap()` with `ok()`? `ok()` just converts `Result` to `Optional`, did you mean to suppress the error?